### DISCUSSION TITLE
May I join the team?

### DISCUSSION THREAD

**Discussion by MudabbirulSaad** [ROM-builders/temporary#19360] [General]
My trees:
Device: https://github.com/MudabbirulSaad/device_oneplus_dumpling (dumpling)
Device: https://github.com/MudabbirulSaad/device_oneplus_cheeseburger (cheeseburger)
Common: https://github.com/MudabbirulSaad/device_oneplus_msm8998-common
Kernel: https://github.com/MudabbirulSaad/kernel_oneplus_msm8998
Vendor: https://github.com/MudabbirulSaad/vendor_oneplus_dumpcheese

my previous attempts:
[#16699](https://github.com/ROM-builders/temporary/pull/16699)
[#18243](https://github.com/ROM-builders/temporary/pull/18243)

both got errors due to sync. 

current attempts:
[#19349](https://github.com/ROM-builders/temporary/pull/19349)
[#19350](https://github.com/ROM-builders/temporary/pull/19350)

I have already learned about git and other necessary stuff to build AOSP.

my tg: https://t.me/mudabbirulsaad

**Comment by Apon77** [comment]
Please follow #learn instructions from the telegram group. Also there is delay in reply because you tagged me, I get notification without tag too.

**Reply by MudabbirulSaad** [reply]
I have checked the #learn note from group. And I know git and the stuffs to build AOSP. I'm maintaining 2 roms for a year. Now what do I needed to do?
